### PR TITLE
ci: add Azure Pipelines 

### PR DESCRIPTION
**Pull request by sergiohgz**
**Type:** chore
**Scope:** ci

The following has been addressed in the PR:

*  There is a related issue? **No**
*  Unit or Functional tests are included in the PR **No**

**Description:** We want to pass tests in all available environments to ensure that everything can work as expected in every platform. Here we add Azure Pipelines to test the execution in Windows, Linux (Ubuntu) and MacOs. 
Also, there are some fixes necessary to work in Windows correctly

### CODE REVIEW

File: core/readme/.prettierrc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "endOfLine": "auto",
```
Comment (juanpicado) [inline core/readme/.prettierrc]: We need this also in the main repo.

**Comment by juanpicado** [comment]
Awesome !!! 👏🏻👏🏻👏🏻👏🏻👏🏻👏🏻👏🏻👏🏻👏🏻

File: core/readme/.prettierrc (hunk 1 above)
Comment (sergiohgz) [reply core/readme/.prettierrc]: I think that when we refactor our eslint config to be more modular and unify there the rules, prettier will contains this for all, included main repo

Comment (juanpicado) [reply core/readme/.prettierrc]: Agreed

**Pull request merged**